python/ray/data/tests/test_formats.py

```diff
@@ -185,12 +186,32 @@ def test_from_torch(shutdown_only, tmp_path):
     torch_dataset = torchvision.datasets.MNIST(tmp_path, download=True)
     expected_data = list(torch_dataset)
 
-    ray_dataset = ray.data.from_torch(torch_dataset)
+    ray_dataset = ray.data.from_torch(torch_dataset, parallelism=1)
```

Comment:
do we need to explicitly specify parallelism here?

Reply:
This test can fail if we have parallelism > 1 since the block order isn't preserved.